docs/cql/time-to-live.rst

```diff
@@ -46,9 +46,20 @@ It’s also possible to set the TTL when performing an INSERT. To do this use:
 
         INSERT INTO heartrate(pet_chip_id, name, heart_rate) VALUES (c63e71f0-936e-11ea-bb37-0242ac130002, 'Rocky', 87) USING TTL 30;
 
-In this case, a TTL of 30 seconds is set. 
+In this case, a TTL of 30 seconds is set for each column. 
 
 
+When you set a TTL with an INSERT operation, the TTL will be applied to each individual column. If you UPDATE a column later without specifying a new TTL, the column's TTL will be updated to either 0 (no expiry) or the table-level TTL. If a column is updated without specifying a TTL, after the TTL that was set for the row during the INSERT operation has elapsed, all columns except the updated one will be expired. However, the row will still be available due to the presence of the unexpired column.
```

Comment:
I am afraid that this paragraph might suggest to the reader a difference between INSERT and UPDATE which doesn't exist: In both INSERT and UPDATE, the given TTL is applied to each individual column **set in this requested**. If an existing column is updated (and both INSERT and UPDATE!) the new TTL (or default TTL, or no expiration) applies to this column.
Although the names INSERT and UPDATE suggests that INSERT inserts a branch-new row and UPDATE modifies an existing row, this needn't be true in Scylla (or Cassandra). INSERT and UPDATE are almost identical (in fact, we call it "upsert" operation), the only different is the so-called **cql row marker** that is added by INSERT by not by UPDATE (we have this document about it: https://docs.scylladb.com/stable/using-scylla/cdc/cdc-basic-operations.html#digression-the-difference-between-inserts-and-updates) but this difference is not relevant to TTL.

It is also important to understand (and explains) that there is not really a notion of a "TTL that was set for the row during the INSERT operation" - INSERT, just like UPDATE, sets a TTL on individual columns and not on the "row". To be more accurate, it does set a TTL also on the "cql row marker" but this is a more subtle effect.